.github/workflows/sycl-nightly.yml

```diff
@@ -244,7 +245,41 @@ jobs:
       sycl_toolchain_artifact: sycl_linux_default
       sycl_toolchain_archive: ${{ needs.ubuntu2204_build.outputs.artifact_archive_name }}
       sycl_toolchain_decompress_command: ${{ needs.ubuntu2204_build.outputs.artifact_decompress_command }}
-      sycl_cts_artifact: sycl_cts_bin
+      sycl_cts_artifact: sycl_cts_bin_linux
+
+  build-sycl-cts-win:
+    needs: build-win
+    if: ${{ always() && !cancelled() && needs.build-win.outputs.build_conclusion == 'success' }}
+    uses: ./.github/workflows/sycl-windows-run-tests.yml
+    with:
+      name: Build SYCL-CTS for Windows
+      runner: '["Windows", "build-e2e"]'
```

Comment:
sorry why are we using `build-e2e` here? i added that for david to test his work. if we want it to mean `fast windows build runner`, thats fine, but just want to be clear

Reply:
Yes, it would be much better to use a fast machine here. Which label should I use instead?

Reply:
thats fine, ill add build-e2e to the fast machines, pr is fine as-is